docker/development/debian/Dockerfile

```diff
@@ -127,19 +151,26 @@ RUN python3.6 -m pip install --upgrade pip \
 
 ## CMake [3.18.4]
```

Comment:
Let's remove the `[version]` since it's now explicit via `ARG`?

```suggestion
## CMake
```

Reply:
I was thinking the same thing :)